testapi.pm

```diff
@@ -949,6 +952,38 @@ sub script_output($;$) {
     return $output;
 }
 
+
+=head2 share_tmp_file
+
+  share_tmp_file($relpath, $content)
+
+Saves content to the file in the worker pool directory using relative path,
+to be fetched via http later on using <autoinst_url>/files/#relpath/#file url.
+Can be used to modify files for specific test needs, e.g. autoinst profiles.
+Dies if cannot open file for writing.
+
+Example:
+  share_tmp_file(autoyast/autoinst.xml, "<profile>Test</profile>")
```

Comment:
Don't you need ticks around the first argument as well? Also I recommend single ticks for the second argument as well

Reply:
fixed